sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/examples/sparql11spec/Section2Test.java

```diff
@@ -88,6 +123,19 @@ public void example_2_3_3() {
 		p();
 	}
 	
+	@Test
+	public void example_2_3_3_model() {
+		String datatype = "specialDatatype";
+		Variable v = query.var(), p = query.var();
```

Comment:
Is v==p or does query.var() return a new object each time?

Reply:
it will return a new `Variable` that is unique to the query instance that it was called on